tests/benchmarks/timing_info/prj_userspace.conf

```diff
@@ -0,0 +1,10 @@
+CONFIG_TEST=y
+CONFIG_EXECUTION_BENCHMARKING=y
+CONFIG_TEST_RANDOM_GENERATOR=y
+CONFIG_HEAP_MEM_POOL_SIZE=256
+CONFIG_MAIN_STACK_SIZE=2048
+CONFIG_FORCE_NO_ASSERT=y
+
+#Disable Userspace
```

Comment:
comment is opposite of what you are doing here. this is redundant anyway since CONFIG_TEST selects CONFIG_TEST_USERSPACE and CONFIG_TEST_HW_STACK_PROTECTION